Makefile

```diff
@@ -0,0 +1,19 @@
+ROOT_DIR:=$(shell dirname $(realpath $(firstword $(MAKEFILE_LIST))))
+
+ifndef CHANNEL
```

Comment:
Actually, CHANNEL is not used anymore. The documentation is outdated. Sorry about that.